### PR TITLE
Add test file

### PR DESCRIPTION
**Pull request by AMill7**
Adding the file with test textes.

Closes https://github.com/githubteacher/sep-22-developers/issues/36

### CODE REVIEW

**Comment by eckstueck** [comment]
:+1:

**Comment by AMill7** [comment]
@eckstueck Welcome!

File: AMill7.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+### Places Visit
+
+
+
+### Emoticons
+  - :tacos:
+  - :beer:
+  - :happy:
+  - :sad:
+
```
Comment (AMill7) [inline AMill7.md:18]: Wrong Emoticons :cry:

**Comment by eckstueck** [comment]
:+1:

**Pull request merged**